hrm-domain/scheduled-tasks/generate-ai-training-set/hrmdbAccess.ts

```diff
@@ -58,7 +58,7 @@ export const streamTrainingSetContacts = async (
 
   const qs = new QueryStream(formattedQuery, [], { highWaterMark: HIGH_WATER_MARK });
   // Expose the readable stream to the caller as a promise for further pipelining
-  const db = await getDbForAccount(accountSid);
+  const db = await Promise.resolve(getDbForAdmin());
```

Comment:
We should think about deleting this task now, I think we're done with it - what do you think @mckinneybock? This was the task that extracted the data and the transcripts and put them in the same file you were using last year

(not for this PR, just thinking)